src/App.js

```diff
@@ -1,21 +1,28 @@
+/* eslint-disable react/jsx-key */
 import React from 'react';
 import {
   Routes,
   Route,
 } from 'react-router-dom';
-import Header from './components/header';
-import BookPage from './components/bookPage';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
+/* eslint-disable-next-line */
+import Header from './components/Header';
+/* eslint-disable-next-line */
+import BookPage from './components/BookPage';
 import Categories from './redux/categories/categories';
```

Comment:
When I try to run the server, the project compiles with errors related to line 12

- [ ] I request that you kindly try to fix this import as seen below 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/155765193-cd227f3e-7c21-4587-b2f4-13cab02f0af6.png)

Reply:
Thank you for your help! All my errors were due to the casing that I changed and this helped me find the source!